src/utilities/methods.js

```diff
@@ -168,3 +169,15 @@ export const formatConditions = (rule) => {
   }
   return Config.emptyFieldValue;
 };
+
+/** The default formatting for dates */
+export const DEFAULT_TIME_FORMAT = 'hh:mm:ss A MM.DD.YYYY';
+
+// Helper to format time in displayable format
+export const formatTime = (value) => {
+  if (value) {
+    const time = moment.utc(value).local();
+    return (time.unix() > 0) ? time.format(DEFAULT_TIME_FORMAT) : '' || Config.emptyFieldValue;
```

Comment:
`Config.emptyFieldValue || ''`

Should this not be the correct order?

Reply:
No, we have view the previous expression holistically. I could add a pair of brackets to make it clear.

`((time.unix() > 0) ? time.format(DEFAULT_TIME_FORMAT) : '') || Config.emptyFieldValue;`

Reply:
Oh, got, my bad.